src/resultsutil.cpp

```diff
@@ -98,16 +111,20 @@ void setupContextMenu(QTreeView* view, CostContextMenu* costContextMenu, int sym
 
         QMenu contextMenu;
         if (callback && symbol.isValid() && actions) {
+            // CHECKME: best would be to check which view we are in and not add the first three items for "Top Hotspots
```

Comment:
isn't that what the `actions` is meant for? i.e. can't we just skip passing the respective action for that view to hide them all?

Reply:
Sounds good. Can you have a look at this?